product/timelines/miq_reports/tl_events_hourly.yaml

```diff
@@ -67,6 +68,7 @@ col_order:
 - container_namespace
 - container_name
 - container_replicator_name
+- middleware_name
```

Comment:
right, that's because the code does now conform to using the explicit field names in event_streams. So you'll see event_streams.middleware_server_name and event_streams.middleware_deployment_name in the db, for example.